prow/config/cache.go

```diff
@@ -121,14 +164,78 @@ func (kp *CacheKeyParts) CacheKey() (CacheKey, error) {
 	return CacheKey(data), nil
 }
 
+func (cache *InRepoConfigCache) handlePresubmit(requestChan chan InrepoconfigPresubmitRequest) {
```

Comment:
nit: maybe change name of requestChan to presubmitChan so it is clear that this is the same chan as presubmitChan in GetPresubmits

Reply:
agree with the point. I still bare with some hope that I can use generic to solve this problem, so would rather leave it as is for now